File: CPP/Program/Software.cpp
#ifndef SOFTWARE_CPP
#define SOFTWARE_CPP

#include <iostream>
#include <string>
using namespace std;

class Software {
private:
    string nama;
    string versi;
    string developer;
    
public:
    Software() {
    }
    
    Software(string nama, string versi, string developer) {
        this->nama = nama;
        this->versi = versi;
        this->developer = developer;
    }
    
    void setNama(string nama) {
        this->nama = nama;
    }
    
    void setVersi(string versi) {
        this->versi = versi;
    }
    
    void setDeveloper(string developer) {
        this->developer = developer;
    }
    
    string getNama() {
        return this->nama;
    }
    
    string getVersi() {
        return this->versi;
    }
    
    string getDeveloper() {
        return this->developer;
    }
    
    ~Software() {
    }
};

#endif // SOFTWARE_CPP

File: Python/Program/Software.py
class Software:
    def __init__(self, nama="", versi="", developer=""):
        self.__nama = nama
        self.__versi = versi
        self.__developer = developer
    
    def set_nama(self, nama):
        self.__nama = nama
    
    def set_versi(self, versi):
        self.__versi = versi
    
    def set_developer(self, developer):
        self.__developer = developer
    
    def get_nama(self):
        return self.__nama
    
    def get_versi(self):
        return self.__versi
    
    def get_developer(self):
        return self.__developer